profile/encode.go

```diff
@@ -493,6 +494,7 @@ var locationDecoder = []decoder{
 		pp.Line = append(pp.Line, Line{})
 		return decodeMessage(b, &pp.Line[n])
 	},
+	func(b *buffer, m message) error { return decodeBool(b, &m.(*Location).IsFolded) },
```

Comment:
Add a comment, similar to other fields above?

Reply:
Done.